plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/ui/PostgreConnectionPage.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.jkiss.dbeaver.ext.postgresql.ui;
 
+import java.util.Locale;
+
```

Comment:
Why did you move it at the beginning of imports?

Reply:
auto import cleanup